lib/src/executables/intl_message_migration.dart

```diff
@@ -199,6 +199,18 @@ void main(List<String> args) async {
     await migratePackage(
         package, packageNameLookup, processedPackages, codemodArgs, dartPaths);
   }
+
+  if (exitCode != 0) {
+    printInBlue(
+        'To resolve these changes, please execute the codemod locally by running :');
+    printInBlue('dart pub global activate --overwrite over_react_codemod');
+    printInBlue(
+        'dart pub global run over_react_codemod:intl_message_migration --migrate-constants --prune-unused');
+  }
+}
+
+void printInBlue(String text) {
+  print('\x1B[34m$text\x1B[0m');
```

Comment:
Surely there's something better than hard-coding the escape sequences. Looks like there might be other things we're using, e.g. https://sourcegraph.plat.workiva.net/github.com/Workiva/over_react/-/blob/tools/analyzer_plugin/tool/init_local_dev.dart?L60:15

Reply:
Agreed, will change to using the io library.